### ISSUE TITLE
Geometry has Z dimension but column does not

### ISSUE THREAD

**Issue by pnoll1** [pnoll1/osmand_map_creation#88]
**Describe the bug**
Program errors out with 
```
psycopg.errors.InvalidParameterValue: Geometry has Z dimension but column does not
```
**To Reproduce**
Steps to reproduce the behavior, Mention an exact address if possible:
1. Use OA global data from 3/16
2. processing.py with batches = ['--normal us:ak us:hi us:mt us:ut us:nv us:az us:nm us:wy us:wa us:or us:id us:co']
3. console will error out with above error from postgis

**Expected behavior**
No errors. 

**Additional context**
Need to find offending file. Does the file have z coordinates? If so, why? Improve error handling and/or logging in merge_oa.

**Comment by pnoll1** [comment]
us/wa/wakiakum-addresses-county.geojson has z data that is mostly, if not all zeros.

```
{"type": "Feature", "properties": {"hash": "2a7175361ea6f935", "number": "34", "street": "BARR ROAD", "unit": "", "city": "ROSBURG", "district": "", "region": "", "postcode": "98643", "id": ""}, "geometry": {"type": "Point", "coordinates": [-123.6361676, 46.32554, 0.0]}}

**Comment by pnoll1** [comment]
fixed in 0b30b4bd8b7aef567e9a0b69199f713a18102998
